docs/faq/install.txt

```diff
@@ -54,6 +54,7 @@ Django version Python versions
 4.0            3.8, 3.9, 3.10
 4.1            3.8, 3.9, 3.10, 3.11 (added in 4.1.3)
 4.2            3.8, 3.9, 3.10, 3.11
+5.0            3.10, 3.11, 3.12
```

Comment:
Is this change part of ticket-34233? (I have no preference 😅)

Reply:
You're picky :wink: I followed the old patterns, see: 

- Django 4.0: https://github.com/django/django/pull/13845/commits/8774b1144c08f18e23381ffae7084dbc05ebfe37
- Django 3.0: https://github.com/django/django/pull/10798/commits/eb0ce6fa3669971f1e21a3fdba270cf33d603eaa
- ...

Reply:
> (I have no preference 😅)

I was only checking. 😜